orchagent/srv6orch.cpp

```diff
@@ -365,18 +382,22 @@ void Srv6Orch::doTaskSidTable(const KeyOpFieldsValuesTuple & tuple)
     SWSS_LOG_ENTER();
     string sid_name = kfvKey(tuple);
     string op = kfvOp(tuple);
-    string sid_list;
+    string sid_list, sidlist_type;
 
     for (auto i : kfvFieldsValues(tuple))
     {
         if (fvField(i) == "path")
         {
           sid_list = fvValue(i);
         }
+        if (fvField(i) == "type")
+        {
+          sidlist_type = fvValue(i);
+        }
```

Comment:
could you update doc/swss-schema.md with the new attribute 'type' in SRV6_SID_LIST_TABLE

Reply:
have updated the swss-schema.md for new attribute 'type' in SRV6_SID_LIST_TABLE

Reply:
@kperumalbfn please help review, thanks.